integration-tests/kyuubi-kubernetes-it/src/test/scala/org/apache/kyuubi/kubernetes/test/deployment/KyuubiOnKubernetesTestsSuite.scala

```diff
@@ -72,10 +74,17 @@ class KyuubiOnKubernetesWithSparkTestsBase extends WithKyuubiServerOnKubernetes
  *  ------------       -------------------------------------------------      ---------------------
  */
 class KyuubiOnKubernetesWithClientSparkTestsSuite
-  extends KyuubiOnKubernetesWithSparkTestsBase {
+  extends KyuubiOnKubernetesWithSparkTestsBase with SparkQueryTests {
   override protected def connectionConf: Map[String, String] = {
-    super.connectionConf ++ Map("spark.submit.deployMode" -> "client")
+    super.connectionConf ++ Map(
+      "spark.submit.deployMode" -> "client",
+      "spark.driver.host" -> kyuubiServerIp,
+      "kyuubi.frontend.connection.url.use.hostname" -> "false")
   }
+
+  override protected def jdbcUrl: String = getJdbcUrl(connectionConf)
+
+  override protected lazy val user: String = "client"
```

Comment:
In this test, Kyuubi use `USER` share level create spark engine. If use same user, kyuubi won't create spark engine in client deploy mode instead of using local mode.

If we use `CONNECTION` share level, will create one spark engine for each sql, which we can't afford.

Reply:
why we do not change user for cluster mode ?

Reply:
We using simpleDFS with cluster mode, we should use default user to avoid conflict of permissions.